deu/atr-received.vbs

```diff
@@ -147,10 +148,12 @@ LOOP UNTIL are_we_passworded_out = false
 ievs_member = trim(ievs_member)
 IEVS_ssn = right(ievs_member, 9)
 IEVS_MEMB_number = left(ievs_member, 2)
-'MsgBox IEVS_MEMB_number
 CALL navigate_to_MAXIS_screen("INFC" , "____")
 CALL write_value_and_transmit("IEVP", 20, 71)
 CALL write_value_and_transmit(IEVS_ssn, 3, 63)
+'checking for NON-DISCLOSURE AGREEMENT REQUIRED FOR ACCESS TO IEVS FUNCTIONS'
```

Comment:
I'm not understanding the err_msg handling for the code on lines 129-160. There is nothing in the do..loop to ensure that an ievs_member has been selected, then the script will end if an err_msg is hit. 

This will need to be updated, or if there is a reason for this coding, comments will need to be added.